user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1180,29 +1259,17 @@ define([
         var path = $target.attr("path");
         var myself = this;
 
-        var url = CONTEXT_PATH + "api/repo/files/" +
-            FileBrowser.encodePathComponents(path == null ? ":" : Encoder.encodeRepositoryPath(path))
-            + "/tree?depth=1&showHidden=" + myself.model.get("showHiddenFiles") + "&filter=*%7CFOLDERS";
+        var url = CONTEXT_PATH + "plugin/scheduler-plugin/api/generic-files/"
+            + FileBrowser.encodePathComponents(encodeGenericPath(path))
+            + "/tree?depth=1&showHidden=" + myself.model.get("showHiddenFiles") + "&filter=FOLDERS";
         $.ajax({
           async: true,
           cache: false, // prevent IE from caching the request
           dataType: "json",
           url: url,
           success: function (response) {
             if (response.children) {
-              var toAppend = "";
-              for(var i = 0; i < response.children.length; i++) {
-                var child = response.children[i].file;
-                toAppend += "<div id=\"" + child.id + "\" class=\"folder\" path=\"" + child.path +
-                    "\" ext=\"" + child.name + "\" desc=\"" + child.name + "\">" +
-                    "<div class=\"element\" role='treeitem' aria-selected='false' aria-expanded='false' tabindex='-1'>" +
-                    "<div class=\"expandCollapse\"></div>" +
-                    "<div class=\"icon\"></div>" +
-                    "<div class=\"title\">" + ( child.title ? child.title : child.name ) + "</div>" +
-                    "</div>" +
-                    "<div class=\"folders\" style=\"\"></div>" +
-                    "</div>"
-              }
+              var toAppend = templates.folders(FileBrowser.fileBrowserModel.get("fileListModel").reformatResponse(response));
               $target.find("> .folders").append(toAppend ? toAppend : "");
               depth = $target.attr("path").split("/").length;
```

Comment:
Can be handled on a follow-up PR.

The `depth` variable is being changed... We should check what this logic is and if it still applies.

Reply:
ah good catch